linera-execution/src/test_utils/mock_application.rs

```diff
@@ -15,6 +15,9 @@ use std::{
     },
 };
 
+#[cfg(web)]
+use js_sys::wasm_bindgen;
```

Comment:
inline this (once) where it's used?

Reply:
Unfortunately the `wasm_bindgen` macro expects the `wasm_bindgen` crate to be available where it's used.